cache.go

```diff
@@ -0,0 +1,19 @@
+package main
```

Comment:
Also you should create this interface in a different package. Say `service/cache`

Reply:
Ok, will do that.